packages/transformers/js/src/JSTransformer.js

```diff
@@ -89,9 +89,9 @@ export default new Transformer({
       }
 
       // Insert node globals
-      if (GLOBAL_RE.test(asset.code)) {
-        walk.ancestor(asset.ast.program, insertGlobals, asset);
-      }
+      // if (GLOBAL_RE.test(asset.code)) {
+      //   walk.ancestor(asset.ast.program, insertGlobals, asset);
+      // }
```

Comment:
Will this be uncommented eventually?

Reply:
yeah... that transform is currently broken